_docs/guid-alias.md

```diff
@@ -2,11 +2,28 @@
 title: GUID Alias
 permalink: /docs/publisher-tutorials/guid-alias/
 ---
-# GUID Alias
-> For Dockstore 1.5.0+
+# Alias
+An **alias** is a unique identifier for a digital entity that can be used as an alternative ID to the one generated by Dockstore. A digital entity refers to a tool, workflow, organization or collection on Dockstore. Aliases are only unique within entity types, so you could use the alias *foobar* for both a tool and an organization. Tools and workflows are treated as a single entity type.
 
-If you would like to setup aliases for your tool or workflow, you can now do so through our API for now. Aliases deal with the situation where there are multiple Dockstore-like sites (including Dockstore), and you have the same tool on each platform. You can use an alias to give this tool a unique name that could potentially be valid across all sites, independent of site specific naming conventions.
+Currently the only way to add an alias to an entity is through our API. You can use our [Swagger UI](https://dockstore.org/api/static/swagger-ui/index.html#) to add an alias.
+
+**Some things to note:**
+* You can create multiple aliases per digital entity, however they must all be unique
+* Aliases are restricted to alphanumerical strings that are case-insensitive and can contain internal hyphens
+* An alias cannot be removed
+
+## Tools and Workflows
+If you would like to setup aliases for your tool or workflow, you can now do so through our API for now. Aliases can deal with the situation where there are multiple Dockstore-like sites (including Dockstore), and you have the same tool on each platform. You can use an alias to give this tool a unique name that could potentially be valid across all sites, independent of site specific naming conventions.
 
 Use the `/entries/{id}/aliases` endpoint to add new aliases. You can specify the aliases as a comma-delimited list.
 
-**Note:** Aliases are restricted to alphanumerical strings that are case-insensitive and can contain internal hyphens.
+## Organizations and Collections
+> For Dockstore 1.6.0+
+
+If you would like to setup aliases for your organization or collection, you must first ensure that the corresponding organization has been approved.
+
+Use the `/organizations/{id}/aliases` endpoint to add an alias for a organization.
+
+Use the `/organizations/collections/{id}/aliases` endpoint to add an alias for a collection.
```

Comment:
I hadn't noticed that in the code this endpoint does not require an organizationId.

This means you can't have an organization named `collections` and more importantly, is inconsistent with other collections APIs.